varya/header.php

```diff
@@ -23,7 +23,7 @@
 <div id="page" class="site">
 	<a class="skip-link screen-reader-text" href="#content"><?php _e( 'Skip to content', 'varya' ); ?></a>
 
-		<header id="masthead" class="site-header default-max-width">
+		<header id="masthead" class="site-header default-max-width <?php if ( has_nav_menu( 'primary' ) ) : echo esc_attr( 'has-main-navigation' ); endif; ?>">
```

Comment:
I think this class would be better on the `<body>` since we already have a `body_class` filter for this kind of thing:

https://github.com/Automattic/themes-workspace/blob/d01dc09e8bffd538514dcf2e713c1d78be34c1f8/varya/inc/template-functions.php#L28-L45

That filter is one of those long standing best practices from `_s` that many Themers are already familiar with.

Reply:
Lines 38-42 are obsolete and can be removed entirely (along with the image_filter function itself).

Reply:
Ah, excellent. Good call. 👍